src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    add: function(number) {
+      return this.result + number;
+    },
+    subtract: function(number) {
+      return this.result - number;
+    },
+    multiply: function(number) {
+      return this.result * number;
+    },
+    divide: function(number) {
+      return this.result / number;
```

Comment:
Instead of using the `function` keyword, you can use the method shorthand syntax for defining object methods.